misc-utils/lsfd-sock-xinfo.c

```diff
@@ -370,10 +370,10 @@ static void load_xinfo_from_proc_unix(ino_t netns_inode)
 		unsigned int st;
 		unsigned long inode;
 		struct unix_xinfo *ux;
-		char path[ sizeof_member(struct unix_xinfo, path) ] = { 0 };
+		char path[UNIX_LINE_LEN] = { 0 };
 
 
-		if (sscanf(line, "%*x: %*x %*x %" SCNx64 " %x %x %lu %s",
+		if (sscanf(line, "%*x: %*x %*x %" SCNx64 " %x %x %lu %[^\n]",
```

Comment:
Cannot we also restrict the size?  It means get `%123[^\n]`, something like:
```C
   "%" stringify_value(UNIX_LINE_LEN) "[^\n]"
```
The problem is if UNIX_LINE_LEN is not a value.

Reply:
Updated. Thank you.